cert/aziot-certd/src/lib.rs

```diff
@@ -214,6 +215,7 @@ fn create_cert<'a>(
 			}
 
 			let mut x509 = openssl::x509::X509::builder().map_err(|err| Error::Internal(InternalError::CreateCert(Box::new(err))))?;
+			x509.set_version(version).map_err(|err| Error::Internal(InternalError::CreateCert(Box::new(err))))?;
```

Comment:
Fix this in `pkcs11-test` too please.

https://github.com/Azure/iot-identity-service/blob/9d5bfea9d6586452993ce9925c260ce3d1249459/pkcs11/pkcs11-test/src/main.rs#L204

```diff
	let mut builder = openssl::x509::X509::builder()?;
+
+	builder.set_version(2)?;
```

Reply:
Added